README.rst

```diff
@@ -150,6 +162,20 @@ And in your ``script.jl``, you can intercept the value with
 
     arg = ARGS[1]  # holds ``"value"``
 
+If you pass only of of them, either options for the executable or arguments to the
+script, you still need to include the separator.
+
+.. code-block:: python
+
+    @python.mark.julia(("--verbose", "--"))  # for options for the executable.
+    def task_func():
+        ...
+
+
+    @python.mark.julia(("--", "value"))  # for arguments for the script.
+    def task_func():
+        ...
+
 
```

Comment:
I would suggest adding an the resulting command that is run by pytask as an example here.